integration-tests/cypress/e2e/project/capabilities/rename.cy.ts

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { Project } from '../../../pages/Project';
+import { Projects } from '../../../pages/Projects';
+import { Flow } from '../../../usecases/Flow';
+
+const projectName = 'Cypress - Disabled Rename Project';
+
+describe('Project download', () => {
+  context('Given a project with a specific name', () => {
+    let projectId: string = '';
+    beforeEach(() => {
+      new Flow().createRobotProject(projectName).then((createProjectData) => {
+        projectId = createProjectData.projectId;
+      });
+    });
+
+    afterEach(() => cy.deleteProject(projectId));
+
+    context('When we try to rename the project from the project browser view', () => {
+      beforeEach(() => new Projects().visit());
+
+      it('Then the rename button should be disabled', () => {
+        new Projects().getActionMenu(projectName).getRenameButton().should('have.class', 'Mui-disabled');
+      });
+    });
+
+    context('When we try to rename the project from the workbench', () => {
+      beforeEach(() => new Project().visit(projectId));
+
+      it('Then the rename button should be disabled', () => {
+        new Project().getProjectNavigationBar(projectName).getRenameButton().should('have.class', 'Mui-disabled');
+      });
+    });
```

Comment:
I need the capabilities to disappear, not be disabled

Reply:
Does the download button disappear too or was it simply disabled? Because it needs to disappear too